Bio/SeqIO/SnapGeneIO.py

```diff
@@ -170,6 +167,16 @@ def _parse_features_packet(length, data, record):
                     qvalues.append(int(value.attributes["int"].value))
             quals[qname] = qvalues
 
+        name = _get_attribute_value(feature, "name")
+        if name:
+            if "label" not in quals:
+                # No explicit label attribute, use the SnapGene name
+                quals["label"] = [name]
```

Comment:
I don't fully understand why you want to do this here. Is this the rare corner case?

Reply:
No, that would be the most common case actually.

To summarize, there are three possible cases:
* _name_ attribute present, _label_ qualifier absent (the most common case, at least in the files I have seen): then we store the name as a _label_ qualifier;
* _name_ attribute and _label_ qualifier both present and containing the same value (the second most common case): then we store that value as a _label_ qualifier;
* _name_ attribute and _label_ qualifier both present and containing different values (the “corner case” that gives issue #3286, and [which seems to occur in ~10% of files](https://github.com/biopython/biopython/issues/3286#issuecomment-697589615)): then we store both values, the name being stored in a non-standard _name_ qualifier.

(Theoretically, there could be a fourth case when no _name_ attribute is present, but I have never seen that and I strongly suspect SnapGene would consider that as invalid; still, I’d rather not assume that a _name_ attribute is always present, hence the initial test above.)

Reply:
OK, and this preserves the current/historic behaviour which mapped the name attribute to the label qualifier.